src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -450,6 +450,9 @@ bool emberAfOperationalCredentialsClusterAddNOCCallback(app::CommandHandler * co
     err = Server::GetInstance().GetFabricTable().Store(fabricIndex);
     VerifyOrExit(err == CHIP_NO_ERROR, nocResponse = ConvertToNOCResponseStatus(err));
 
+    // Notify the secure session of the new fabric.
+    commandObj->GetExchangeContext()->GetSessionHandle()->AsSecureSession()->NewFabric(commandObj->GetAccessingFabricIndex());
```

Comment:
If someone set up ACLs allowing AddNOC as a group command (is that possible?) this will crash.  Might be better to just error out if the session is not a secure session.

Reply:
We don't allow commissioning over group sessions but I can try to add more checks for that.

Reply:
Again though, note we have that kind of code in a few other places in this file.

Reply:
What kind of code?  The kind that needs to get fixed to resolve #13397?

Reply:
The entire chain of `commandObj->GetExchangeContext()->GetSessionHandle()->AsSecureSession()` occurs in a few places in this cluster implementation, so it's an assumption. It should be questioned and audited of course, so I agree, #13397 needs to be addressed.

Reply:
AddNOC requires Administer privilege, and Administer ACLs cannot be Group AuthMode, so per spec we should never be in this command as a group session.

Reply:
> so it's an assumption

It's an assumption that was added recently, and not necessarily correctly...

Thank you for the ACL check bits.  Can we please add a comment on the AsSecureSession() call explaining why it's safe, with reference to that?